### ISSUE TITLE
Source code not detected in multi-module Maven project

### ISSUE THREAD

**Issue by antjori** [felipebz/sonar-plsql#24]
Hey Felipe,

First of all, I'd like to thank you for your work! :+1: 

I have a multi-module Maven project where one of them is fully composed by SQL. Another one is only composed by Java.

After putting my SonarQube analysing the Java part, I wanted to do the same for SQL and I've adopted your plugin for it.

The Java analysis works properly whereas the SQL does not. Am I missing any configuration? Should I explicitly tell your SQL Sonar plugin where to look for the SQL sources?

Regards.

**Comment by felipebz** [comment]
Hi @antjori.

I have a similar setup here, some SQL files are inside the directory src\main\plsql in a separate module. You need to tell the SonarQube Scanner for Maven to include your SQL files in the analysis.

In the pom.xml, you can add something like:
```xml
<properties>
    <sonar.sources>src\main\plsql</sonar.sources>
    <sonar.tests>src\test\plsql</sonar.tests>
</properties>
```

I hope this helps.

**Comment by antjori** [comment]
Hello Felipe,

Thank you so much for your suggestion. It did work! :smiley:

Regards.
